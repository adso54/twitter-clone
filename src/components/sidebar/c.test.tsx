import { render } from '@testing-library/react';
import '@testing-library/jest-dom'
import Sidebar from './index'

test('Rendered Sidebar match the snapshot', async () => {
    const component = render(Sidebar())
    expect(component).toMatchSnapshot()
})