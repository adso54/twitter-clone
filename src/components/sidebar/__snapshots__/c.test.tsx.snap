// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendered Sidebar match the snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav
        class="sc-bczRLJ eQeEmt"
      >
        <div
          class="sc-gsnTZi bUtEwk"
        >
          <div
            class="sc-dkzDqf guXRpg"
          >
            <div
              class="sc-iBkjds iXbJlf"
            >
              <svg>
                logo.svg
              </svg>
            </div>
          </div>
          <div
            class="sc-eCYdqJ huTjpC"
          >
            <div
              class="sc-gKXOVf uFCtg"
            >
              <svg>
                house.svg
              </svg>
            </div>
            <div
              class="sc-jSMfEi dehjIL"
            >
              Home
            </div>
          </div>
          <div
            class="sc-eCYdqJ huTjpC"
          >
            <div
              class="sc-gKXOVf uFCtg"
            >
              <svg>
                magnifier.svg
              </svg>
            </div>
            <div
              class="sc-jSMfEi dehjIL"
            >
              Search
            </div>
          </div>
          <div
            class="sc-eCYdqJ huTjpC"
          >
            <div
              class="sc-gKXOVf uFCtg"
            >
              <svg>
                bell.svg
              </svg>
            </div>
            <div
              class="sc-jSMfEi dehjIL"
            >
              Notifications
            </div>
          </div>
          <div
            class="sc-eCYdqJ huTjpC"
          >
            <div
              class="sc-gKXOVf uFCtg"
            >
              <svg>
                letter.svg
              </svg>
            </div>
            <div
              class="sc-jSMfEi dehjIL"
            >
              Messages
            </div>
          </div>
          <div
            class="sc-iqcoie eOUlpy"
          >
            <div
              class="sc-crXcEl gGjUQO"
            >
              <svg>
                feather.svg
              </svg>
            </div>
            <div
              class="sc-evZas eRbBoj"
            >
              Tweet
            </div>
          </div>
          <div
            class="sc-hKMtZM gmrExI"
          >
            <div
              class="sc-eCYdqJ huTjpC"
            >
              <div
                class="sc-gKXOVf uFCtg"
              >
                <svg>
                  user.svg
                </svg>
              </div>
              <div
                class="sc-jSMfEi dehjIL"
              >
                Your Name
              </div>
            </div>
          </div>
        </div>
      </nav>
    </div>
  </body>,
  "container": <div>
    <nav
      class="sc-bczRLJ eQeEmt"
    >
      <div
        class="sc-gsnTZi bUtEwk"
      >
        <div
          class="sc-dkzDqf guXRpg"
        >
          <div
            class="sc-iBkjds iXbJlf"
          >
            <svg>
              logo.svg
            </svg>
          </div>
        </div>
        <div
          class="sc-eCYdqJ huTjpC"
        >
          <div
            class="sc-gKXOVf uFCtg"
          >
            <svg>
              house.svg
            </svg>
          </div>
          <div
            class="sc-jSMfEi dehjIL"
          >
            Home
          </div>
        </div>
        <div
          class="sc-eCYdqJ huTjpC"
        >
          <div
            class="sc-gKXOVf uFCtg"
          >
            <svg>
              magnifier.svg
            </svg>
          </div>
          <div
            class="sc-jSMfEi dehjIL"
          >
            Search
          </div>
        </div>
        <div
          class="sc-eCYdqJ huTjpC"
        >
          <div
            class="sc-gKXOVf uFCtg"
          >
            <svg>
              bell.svg
            </svg>
          </div>
          <div
            class="sc-jSMfEi dehjIL"
          >
            Notifications
          </div>
        </div>
        <div
          class="sc-eCYdqJ huTjpC"
        >
          <div
            class="sc-gKXOVf uFCtg"
          >
            <svg>
              letter.svg
            </svg>
          </div>
          <div
            class="sc-jSMfEi dehjIL"
          >
            Messages
          </div>
        </div>
        <div
          class="sc-iqcoie eOUlpy"
        >
          <div
            class="sc-crXcEl gGjUQO"
          >
            <svg>
              feather.svg
            </svg>
          </div>
          <div
            class="sc-evZas eRbBoj"
          >
            Tweet
          </div>
        </div>
        <div
          class="sc-hKMtZM gmrExI"
        >
          <div
            class="sc-eCYdqJ huTjpC"
          >
            <div
              class="sc-gKXOVf uFCtg"
            >
              <svg>
                user.svg
              </svg>
            </div>
            <div
              class="sc-jSMfEi dehjIL"
            >
              Your Name
            </div>
          </div>
        </div>
      </div>
    </nav>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
